src/components/accessories/table/Table.tsx

```diff
@@ -0,0 +1,179 @@
+    const isAsc = orderBy === property && order === 'asc';
+    setOrder(isAsc ? 'desc' : 'asc');
+    setOrderBy(property);
+  };
+
+  const renderIcon = (type: TActions) => {
+    switch(type){
+      case 'edit':
+        return (
+          <IconButton size="small" onClick={onEdit}>
+            <Edit />
+          </IconButton>
+        )
+      case 'delete':
+        return (
+          <IconButton size="small" onClick={onDelete}>
+            <Delete color="secondary" />
+          </IconButton>
+        )
+      case 'print':
+        return (
+          <IconButton size="small" onClick={onPrint}>
+            <Print color="primary" />
+          </IconButton>
+        )
+    }
+  }
+
+  const renderActions = () => {
+    if (onEdit || onDelete || onPrint || onView) {
+      return (
+        <TableCell scope="row" align="right" size="small" style={{ minWidth: 125 }}>
+          {(onEdit) ? renderIcon('edit') : ''}
+          {(onPrint) ? renderIcon('print') : ''}
+          {(onDelete) ? renderIcon('delete') : ''}
+        </TableCell>
+      );
+    }
+  }
+
+  return (
+    <>
+      <TableContainer component={Paper}>
+        <MaterialComponent className="table" aria-label="simple table">
+          <TableHead className="table_header">
+            <TableRow>
+              {(isCollapsabile) ? <TableCell /> : ''}
+              {tableHeader.map((h: string, i) => (
+                <TableCell key={i}>
+                  {(columnsOrder.includes(h)) ?
+                    <TableSortLabel
+                      active={orderBy === h}
+                      direction={orderBy === h ? order : 'asc'}
+                      onClick={createSortHandler(h)}
+                    >
+                      {labelData[h]}
+                    </TableSortLabel> : labelData[h]}
+                </TableCell>
+              ))}
+              <TableCell>&nbsp;</TableCell>
+            </TableRow>
+          </TableHead>
+          <TableBody className="table_body">
+            {stableSort(rowData, getComparator(order, orderBy))
+              .slice(page * rowsPerPage, page * rowsPerPage + rowsPerPage)
+              .map((row, index) => (
+                <TableBodyRow
+                  row={row}
+                  rowIndex={index}
+                  labelData={labelData}
+                  tableHeader={tableHeader}
+                  renderActions={renderActions}
+                  isCollapsabile={isCollapsabile}
+                />
+              ))}
+          </TableBody>
+        </MaterialComponent>
+      </TableContainer>
+      {(rowData.length > rowsPerPage) ?
+        <TablePagination
+          component="div"
+          count={rowData.length}
+          rowsPerPage={rowsPerPage}
+          rowsPerPageOptions={[rowsPerPage]}
+          page={page}
+          onChangePage={handleChangePage}
+        /> : ''}
+    </>
+  );
+}
+
+export default Table;
+
+const TableBodyRow: FunctionComponent<IRowProps> = ({
```

Comment:
I think it's a good idea to keep one component per module. Perhaps we should have something like:

`accessories/table/Table.tsx`
`accessories/table/TableBodyRow.tsx`